### PR TITLE
Instruct beginners to start minikube cluster with proxy

### PR DESCRIPTION
**Pull request by leonliao**
Beginners don't know the details for troubleshooting, better instruction will save much time for beginners to troubleshoot logs in minikube complaining gcr.io not reachable and etc.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2250)

### CODE REVIEW

**Comment by leonliao** [comment]
I signed it!

File: docs/tutorials/stateless-application/hello-minikube.md [hunk 1]
```diff
@@ -70,12 +70,23 @@ curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s htt
...
+curl https://cloud.google.com/container-registry/ 
+```
 
-Start the Minikube cluster:
+If NO proxy is required, start the Minikube cluster: 
 
 ```shell
 minikube start --vm-driver=xhyve
 ```
+If a proxy server is required, use below way to start Minikube cluster with proxy setting:
```
Comment (jaredbhatti) [inline docs/tutorials/stateless-application/hello-minikube.md]: Suggest replacing "use below way" to "use the following method"

Comment (leonliao) [inline docs/tutorials/stateless-application/hello-minikube.md]: OK, done.

**Comment by chenopis** [comment]
@leonliao could you make one more small change and turn the URL [https://cloud.google.com/container-registry/](https://cloud.google.com/container-registry/) in the sentence into a link? Thx

**Comment by jaredbhatti** [comment]
I merged the change and added the link in manually.

**Comment by chenopis** [comment]
@jaredbhatti thx!

**Comment by leonliao** [comment]
@chenopis @jaredbhatti Thanks for the change. Was on holiday for last week, didn't check mail.

**Comment by chenopis** [comment]
@leonliao No prob. Thanks for your PR!

**Pull request merged**